src/api/db/structure.sql

```diff
@@ -263,7 +263,7 @@ CREATE TABLE `bs_requests` (
 
 CREATE TABLE `cache_lines` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `key` varchar(4096) COLLATE utf8_unicode_ci NOT NULL,
+  `key` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
```

Comment:
There is no validation for this constraint in the [CacheLines](https://github.com/evanrolfe/open-build-service/blob/master/src/api/app/models/cache_line.rb) model but I don't see this causing any major problems at the moment.